ember-cli-build.js

```diff
@@ -18,6 +18,10 @@ module.exports = function (defaults) {
     'ember-cli-babel': {
       includePolyfill: true,
     },
+
+    'ember-cli-string-helpers': {
+      only: ['classify'],
```

Comment:
Should we not just create a `classify` helper in the dummy app and remove the addon completely? https://api.emberjs.com/ember/release/classes/String/methods/loc?anchor=classify

Reply:
That makes too much sense :smile: Done